### DISCUSSION TITLE
[A-L] (2023/24) Lezione 8 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#176] [Esercizi - Viaggi/Piazza]
Dimostrare la seguente proprietà sull'anello $(\mathbb{Z},+,\cdot)$:
- $MCD(a,0)=|a|$

**Comment by Elia-Belli** [comment]
- Per definizione di $MCD$, $(a,b) \neq 0 \Rightarrow a\neq0 \Rightarrow |a|\geq 1$
- $(|a|)|a$, infatti possiamo scrivere $a=|a|q$, con $q=1$ se $a>0$, e $q=-1$ se $a<0$
- $(|a|)|0$, infatti possiamo scrivere $0=|a|q$, con $q=0$
Possiamo scrivere $|a|=ax_0+0y_0$, con $x_0=1$ se $a>0$, e $x_0=-1$ se $a<0$
- Sia $d': d'|a\ \land\ d'|0 \Rightarrow d'|as+0t,\ \forall s,t \Rightarrow d'|ax_0+0y_0 \Rightarrow d'|(|a|)$

$\Rightarrow MCD(a,0)=|a|$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/c25bdd50-6b4f-4263-bc30-d05bef53d01e)
